<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="1" ignored="0" total="4" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2019-09-16T15:31:22 EDT" name="suiteOfSuites" finished-at="2019-09-16T15:31:22 EDT" duration-ms="0">
    <groups>
    </groups>
  </suite> <!-- suiteOfSuites -->
  <suite started-at="2019-09-16T15:31:20 EDT" name="SuiteOne" finished-at="2019-09-16T15:31:21 EDT" duration-ms="1413">
    <groups>
      <group name="testOne">
        <method signature="ClassOne.hello1()[pri:0, instance:pageTest.weCare.ClassOne@3cc2931c]" name="hello1" class="pageTest.weCare.ClassOne"/>
        <method signature="ClassOne.hello2()[pri:1, instance:pageTest.weCare.ClassOne@3cc2931c]" name="hello2" class="pageTest.weCare.ClassOne"/>
        <method signature="ClassOne.hello3()[pri:2, instance:pageTest.weCare.ClassOne@3cc2931c]" name="hello3" class="pageTest.weCare.ClassOne"/>
      </group> <!-- testOne -->
      <group name="regression">
        <method signature="ClassOne.hello1()[pri:0, instance:pageTest.weCare.ClassOne@3cc2931c]" name="hello1" class="pageTest.weCare.ClassOne"/>
        <method signature="ClassOne.hello2()[pri:1, instance:pageTest.weCare.ClassOne@3cc2931c]" name="hello2" class="pageTest.weCare.ClassOne"/>
        <method signature="ClassOne.hello3()[pri:2, instance:pageTest.weCare.ClassOne@3cc2931c]" name="hello3" class="pageTest.weCare.ClassOne"/>
      </group> <!-- regression -->
    </groups>
    <test started-at="2019-09-16T15:31:20 EDT" name="TestOne" finished-at="2019-09-16T15:31:21 EDT" duration-ms="1413">
      <class name="pageTest.weCare.ClassOne">
        <test-method is-config="true" signature="setupDriver(java.lang.String)[pri:0, instance:pageTest.weCare.ClassOne@3cc2931c]" started-at="2019-09-16T15:31:20 EDT" name="setupDriver" finished-at="2019-09-16T15:31:20 EDT" duration-ms="16" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupDriver -->
        <test-method is-config="true" signature="setupExtentReport()[pri:0, instance:pageTest.weCare.ClassOne@3cc2931c]" started-at="2019-09-16T15:31:21 EDT" name="setupExtentReport" finished-at="2019-09-16T15:31:21 EDT" duration-ms="126" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupExtentReport -->
        <test-method signature="hello1()[pri:0, instance:pageTest.weCare.ClassOne@3cc2931c]" started-at="2019-09-16T15:31:21 EDT" name="hello1" finished-at="2019-09-16T15:31:21 EDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- hello1 -->
        <test-method retried="true" signature="hello2()[pri:1, instance:pageTest.weCare.ClassOne@3cc2931c]" started-at="2019-09-16T15:31:21 EDT" name="hello2" finished-at="2019-09-16T15:31:21 EDT" duration-ms="0" status="SKIP">
          <exception class="junit.framework.AssertionFailedError">
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:48)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at pageTest.weCare.ClassOne.hello2(ClassOne.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1175)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- hello2 -->
        <test-method signature="hello2()[pri:1, instance:pageTest.weCare.ClassOne@3cc2931c]" started-at="2019-09-16T15:31:21 EDT" name="hello2" finished-at="2019-09-16T15:31:21 EDT" duration-ms="0" status="FAIL">
          <exception class="junit.framework.AssertionFailedError">
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:48)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at pageTest.weCare.ClassOne.hello2(ClassOne.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:204)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1175)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- hello2 -->
        <test-method signature="hello3()[pri:2, instance:pageTest.weCare.ClassOne@3cc2931c]" started-at="2019-09-16T15:31:21 EDT" name="hello3" finished-at="2019-09-16T15:31:21 EDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- hello3 -->
        <test-method is-config="true" signature="tearDownExtentReport()[pri:0, instance:pageTest.weCare.ClassOne@3cc2931c]" started-at="2019-09-16T15:31:21 EDT" name="tearDownExtentReport" finished-at="2019-09-16T15:31:21 EDT" duration-ms="551" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownExtentReport -->
      </class> <!-- pageTest.weCare.ClassOne -->
    </test> <!-- TestOne -->
  </suite> <!-- SuiteOne -->
</testng-results>
